tutorials/sem-terraform/index.mdx

```diff
@@ -22,7 +22,7 @@ In this tutorial, you will use a Terraform configuration file that builds your e
 <Message type="requirement">
  - You have an account and logged into the [Scaleway console](https://console.scaleway.com/)
  - You have created an [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
- - You have [configured your environment variables](/developer-tools/devtools/reference-content/environment-variables/#recommended-scaleway-configuration-file)
+ - You have [configured your environment variables](/developer-tools/scaleway-cli/reference-content/environment-variables/#recommended-scaleway-configuration-file)
```

Comment:
Environment variables can be shared between different devtools. Will you duplicate the content for all the tools?